docs/ide/installation.rst

```diff
@@ -0,0 +1,14 @@
+.. _ide_installation:
```

Comment:
I think we do not need an extra file for installation.
I would put it all inside `ide/index.rst`